src/components/Theme/src/default-theme.js

```diff
@@ -28,7 +33,7 @@ export default function defaultTheme() {
 		button: {
 			size: 'medium',
 			variant: 'primary',
-			shape: 'rounded',
+			shape: undefined,
```

Comment:
I like the convention of having even the props with `undefined` as their default because it makes `default-theme.js` a good quick reference for which props are themeable on each component without having to look through each component.